### PR TITLE
Update Algothrithmus for calculated Banking interval

### PR DESCRIPTION
**Pull request by TruongQuangSB**
- Update Algothrithmus for Banking interval with Banking line of form : RAMPE_BLOSS & RAMPE_S

### CODE REVIEW

File: java/bundles/org.eclipse.set.ppmodel.extensions/src/org/eclipse/set/ppmodel/extensions/BankingInterval.xtend [hunk 1]
```diff
@@ -93,87 +91,16 @@ class BankingInterval extends ValueInterval<BigDecimal, BigDecimal> {
...
 		DirectedEdgePath<TOP_Kante, TOP_Knoten, Punkt_Objekt_TOP_Kante_AttributeGroup> path,
-		Punkt_Objekt_TOP_Kante_AttributeGroup singlePoint
+		Ueberhoehungslinie bankingLine
 	) {
-		return BigDecimal.valueOf(
-			path.distance(path.start, singlePoint)
-		)
-	}
-
-	private static class BankRouting implements Routing<TOP_Kante, TOP_Knoten, Punkt_Objekt_TOP_Kante_AttributeGroup> {
```
Comment (TruongQuangSB) [inline java/bundles/org.eclipse.set.ppmodel.extensions/src/org/eclipse/set/ppmodel/extensions/BankingInterval.xtend:105]: was moved to BankRouting.xtend

**Review by pstuecker — APPROVED** [review]

**Pull request merged**